base/markdown/GitHub/GitHub.jl

```diff
@@ -33,7 +33,83 @@ function github_paragraph(stream::IO, md::MD, config::Config)
     return true
 end
 
-# TODO: tables
+typealias Row Vector
 
-@flavor github [list, indentcode, blockquote, fencedcode, hashheader, github_paragraph,
-                 en_dash, inline_code, asterisk_bold, asterisk_italic, image, link]
+type Table
+    rows::Vector{Row}
+    align
+end
+
+function github_table(stream::IO, md::MD, config::Config)
+    withstream(stream) do
+        rows = Row[]
+        n = 0
+        align = :r # default is to align right
```

Comment:
Using colours is a good idea. I think that can probably fixed in the terminal rendering code so I'll take a look at that.